landoapi/api/transplants.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Transplant API
+See the OpenAPI Specification for this API in the spec/swagger.yml file.
+"""
+from connexion import problem
+from landoapi.models.transplant import (
+    Transplant,
+    TransplantNotCreatedException,
+    TransplantNotFoundException,
+    TransplantRevisionNotFoundException,
+)
+
+
+def land(revision_id, api_key=None):
+    """ API endpoint at /revisions/{id}/transplants to land revision. """
+    try:
+        transplant = Transplant.create(revision_id, api_key)
+    except TransplantRevisionNotFoundException:
+        # We could not find a matching revision.
+        return problem(
+            404,
+            'Revision not found',
+            'The requested revision does not exist',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
+        )
+    except TransplantNotCreatedException:
+        # We could not find a matching revision.
+        return problem(
+            502,
+            'Transplant not created',
+            'The requested revision does exist, but landing failed',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502'
+        )
+
+    return {'id': transplant.id}, 202
+
+
+def check_status(transplant_id):
+    """ API endpoint at /revisions/{id}/transplants to list landing jobs
```

Comment:
@purelogiq I moved the design bit to https://public.etherpad-mozilla.org/p/lando-api-urls